config/locales/en.yml

```diff
@@ -20,4 +20,160 @@
 # available at http://guides.rubyonrails.org/i18n.html.
 
 en:
-  hello: "Hello world"
+  full_title: "Open Source Friday | This Friday, invest a few hours contributing to the software you use and love."
+  Open Source Friday: "Open Source Friday"
+  full_description: "Open source is made by people just like you. This Friday, invest a few hours contributing to the software you use and love."
+  Contributors: "Contributors"
+  Businesses: "Businesses"
+  Maintainers: "Maintainers"
+  headline:
+    businesses: "Your business runs on open source. Learn how to get involved."
+    maintainers: "Give your project some love to help others contribute where you need it most."
+    contributors: "Learn what it means to contribute, and how to find and get started on a new project."
+  Sign up with GitHub: "Sign up with GitHub"
+  Sign up: "Sign up"
+  Sign Out: "Sign out"
+
+  # app/views/home/index.html.erb
+  index:
+    main:
+      title: "Contribute on Open Source Friday"
+      contribution: "Learn to contribute"
+      link_html: "<a class=\"\" href=\"https://opensource.guide/how-to-contribute/\">Read the guide%{icon}</a>"
```

Comment:
@miurahr Ok, no worries. Yeh, URL/name parameters would be good and if we could use `link_to` or similar that would be great.